six/modules/c++/six.sicd/include/six/sicd/Utilities.h

```diff
@@ -329,6 +329,14 @@ class Utilities
             const std::vector<std::string>& schemaPaths =
                     std::vector<std::string>(),
             logging::Logger* logger = NULL);
+
+    /*!
+     * Create a fake SICD that's populated enough for
+     * general testing code to run without throwing exceptions
+     *
+     * \return mock ComplexData object
+     */
+    static ComplexData createFakeComplexData();
```

Comment:
Done. I was thinking all our compilers would do NRVO here so the copy wouldn't actually happen, but it looks like Solaris doesn't....